hack/generate/olm_bindata.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+function get_olm_manifests() {
+    mkdir olm-manifests
+    echo "downloading olm manifests for version ${1}"
+    wget -O olm-manifests/olm.yaml "https://github.com/operator-framework/operator-lifecycle-manager/releases/download/${1}/olm.yaml"
+    wget -O olm-manifests/crds.yaml "https://github.com/operator-framework/operator-lifecycle-manager/releases/download/${1}/crds.yaml"
+}
+
+function remove_olm_manifests {
+    rm -rf olm-manifests
+}
+
+# check for files starting with the name "olm-bindata" inside internal/olm folder
+function delete_old_olmbindata {
+    echo "Deleting previous versions of olm-bindata files if they exist"
+    find internal/bindata/olmbindata -maxdepth 1 -type f -name manifests-* -exec rm {} \;
+}
+
+# TODO:
+# 1. Modify this to accept multiple versions and download bindata.
+# 2. Discuss on the number of olm versions of will be supported.
+FILE=internal/bindata/olmbindata/"manifests-"$1.go
```

Comment:
I'd still prefer `internal/bindata/olm`, since we already know this is the bindata dir.